assets/src/application/ui/layout/entityList/filterBar/style.scss

```diff
@@ -114,10 +118,6 @@ $filter-btn: '.esprs-button.ant-btn.ee-filter-bar__btn';
 			min-width: calc(var(--ee-font-size-default) * 9);
 		}
 
-		&--big {
-			min-width: calc(var(--ee-font-size-default) * 15);
-		}
-
 		&--bigger {
 			min-width: calc(var(--ee-font-size-default) * 21);
 		}
```

Comment:
why did you take this one filter size out?

the intend was for these to be available for use in the future when styling other filter bars

Reply:
:man_facepalming: I was in the cleaning mood;
will add back in the next PR

Reply:
beat ya to it 😆 

(pushed directly to EDTR/master)